x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout.tsx

```diff
@@ -57,6 +58,8 @@ const FlyoutFooterWPadding = styled(EuiFlyoutFooter)`
   padding: 16px 24px !important;
 `;
 
+const MAX_VIEW_AGENTS_COUNT = 250;
```

Comment:
I didn't expect failing already over 250, but if you tested it it should be ok

Reply:
Looks like we do not run kibana the same way in dev and prod and in production it seems safe to go with 2000 agents will update the PR 
https://github.com/elastic/kibana/blob/ebe0d1b0bfe22eddae16d67ab3bd9588b1415908/src/dev/build/tasks/bin/scripts/kibana#L29